examples/ex37.cpp

```diff
@@ -180,16 +133,18 @@ int main(int argc, char *argv[])
    int ref_levels = 5;
    int order = 2;
    real_t alpha = 1.0;
+   real_t growth = 2;
    real_t epsilon = 0.01;
    real_t vol_fraction = 0.5;
    int max_it = 1e3;
-   real_t itol = 1e-1;
+   real_t itol = 1e-2;
    real_t ntol = 1e-4;
    real_t rho_min = 1e-6;
    real_t lambda = 1.0;
    real_t mu = 1.0;
    bool glvis_visualization = true;
    bool paraview_output = false;
+   bool newton = true;
```

Comment:
Suggestion: Use a string such as `root_finding_alg` that can be either `newton` or `bisection`. It is not clear until reading on what `newton` is used for

Reply:
See also https://github.com/mfem/mfem/pull/4645#discussion_r1898110950